CHANGELOG.txt

```diff
@@ -1,3 +1,15 @@
+Version 3.3.4
+====
+
+Last call for Python2 support.
+
+* #850 Fix socket.error is a deprecated alias of OSError used on Python versions lower than 3.3.
+* #846 Use inter-process mutex to prevent concurrent neoVI device open.
```

Comment:
I see this in the change log here but https://github.com/hardbyte/python-can/pull/621 is not included in this release branch

Reply:
Ahh my mistake. I'll remove the line from the changelog as I closed that issue without backporting.

Reply:
Can we include the change in 3.3.4?

Reply:
Sure I'm happy to merge it in if you want to open a PR against this `release-3.3.4` branch

Reply:
Please see https://github.com/hardbyte/python-can/pull/903